evfl/actor.py

```diff
@@ -15,8 +15,10 @@ def __init__(self) -> None:
         self.actions: typing.List[StringHolder] = []
         self.queries: typing.List[StringHolder] = []
         self.params: typing.Optional[Container] = None
-        # TODO: investigate what this is (always 1 for flowcharts, but sometimes > 1 for timelines)
-        self.x36: int = 0xffff
+        # lists how many currently-playing clips an actor can be part of at the same time
```

Comment:
Can we mention that this is always 1 for flowcharts?

Reply:
Sure. New commit has it taken care of